de.tudarmstadt.ukp.dkpro.argumentation.nio/src/main/java/de/tudarmstadt/ukp/dkpro/argumentation/annotations/AnnotatedDocument.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (c) the Department of Informatics, Technische Universität Darmstadt. All Rights Reserved.
+ *
+ * Unauthorized distribution of this file via any medium is strictly prohibited.
+ */
+package de.tudarmstadt.ukp.dkpro.argumentation.annotations;
```

Comment:
What is the purpose of classes in this package and what is the difference from the UIMA-based annotations?